### ISSUE TITLE
Two notices raised

### ISSUE THREAD

**Issue by alexkuc** [lucatume/wp-browser#281]
Hello!

When running wpunit/integration tests, I get two notices:

```
Notice: Trying to get property 'domain' of non-object in /Applications/MAMP/htdocs/king-solomon/vendor/lucatume/wp-browser/src/includes/factory/class-wp-unittest-factory-for-blog.php on line 9

Notice: Trying to get property 'id' of non-object in /Applications/MAMP/htdocs/king-solomon/vendor/lucatume/wp-browser/src/includes/factory/class-wp-unittest-factory-for-blog.php on line 12
```

Using latest (`2.2.16`) WPBrowser which was installed via Composer. PHP version is `PHP 7.2.14 (cli)`.

I am not sure if I am doing something wrong on my end or not.

**Comment by lucatume** [comment]
Hi,

Thanks for reporting the issue, this does not seem related to anything you are doing but, to make sure, could you post your Codeception configuration file and suite configuration file?
Are you using the blog factory in any capacity in at least one test? If so: could you please post a minimal test example?

**Comment by lucatume** [comment]
@alexkuc you were not doing anything wrong: I've found out I had introduced this issue in `2.2.16` and have fixed this in `2.2.17`.
